modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,320 @@
+However, since the Sync Gateway contains sensitive information, it is recommended that you create a secret from the config file and pass that to the Sync Gateway.
+Run the following command to create a secret called "sgw-config" or "sgw-config-import" with the corresponding Sync Gateway configuration file.
++
+[source,console]
+----
+kubectl create secret generic [sgw-config|sgw-config-import] --from-file [configuration-file]
+----
++
+If successful, you will see the following.
++
+[source,console]
+----
+secret "sgw-config" created
+----
+
+=== Deployment controller
+
+In this section you will deploy the Sync Gateway cluster with the configuration file that you created above.
+
+You will be using a https://kubernetes.io/docs/concepts/workloads/controllers/deployment/[Kubernetes deployment controller].
+A deployment controller allows you to define the number of Sync Gateway replicas and other parameters.
+
+. Open the controller file corresponding to your deployment.
++
+|===
+| Regular Node | Importer Node
+
+| link:{attachmentsdir}/kubernetes/sgw-deployment.yaml[sgw-deployment.yaml]
+| link:{attachmentsdir}/kubernetes/sgw-deployment-import.yaml[sgw-deployment-import.yaml]
+|===
+The following example shows the controller file for a regular node.
++
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: sync-gateway # <1>
+spec:
+  replicas: 2 # <2>
+  template:
+    metadata:
+      labels:
+        app: sync-gateway
+    spec:
+      containers:
+        - name: sync-gateway
+          image: couchbase/sync-gateway:2.1.2-enterprise # <3>
+          args: ["/sync-gateway-config/sgw-config.json"] # <4>
+          volumeMounts: # <5>
+            - name: sgw-config-volume
+              mountPath: /sync-gateway-config
+              readOnly: true
+          env:
+            - name: GOMAXPROCS # <6>
+              value: "2"
+          resources:
+            requests:
+              cpu: "2"
+            limits:
+              cpu: "2" # <7>
+      volumes: # <8>
+        - name: sgw-config-volume
+          secret:
+            secretName: sgw-config
+----
+<1> `metadata.name`: The name of the deployment is "sync-gateway".
+<2> `spec.replicas`: 2 Sync Gateway replicas that are deployed at most.
+* For import node deployment, this *must* be 1 because there can be at most 1 Sync Gateway replicator in a cluster.
+If high availability is important, you probably want at least 2 Sync Gateway import nodes.
+* For replicator node deployment, this *must* be 1 because there can be at most 1 Sync Gateway import node in a cluster.
+<3> `containers[].image`: Points to the docker image for Sync Gateway.
+<4> `containers[].args`: Points to the Sync Gateway configuration file named "sgw-config-working.json" which is mounted at the path specified via the `volumeMounts` config.
```

Comment:
@jamiltz 
```suggestion
<4> `containers[].args`: Points to the Sync Gateway configuration file named "sgw-config-working.json" which is mounted at the path specified via the `volumeMounts` config.
* If you are using the sample config files, this would be "sgw-import-config-working.json" for the import node
```

Reply:
```suggestion
<4> `containers[].args`: Points to the Sync Gateway configuration file named "sgw-config-working.json" which is mounted at the path specified via the `volumeMounts` config.
If you are using the sample config files, this would be "sgw-config-import.json" for the import node.
```